target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1210-12.dts

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
These tend to be dual licensed, with also MIT, do we not want this here as well?

Reply:
Will do.